.github/workflows/changesets.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    if: ${{ github.repository_owner == 'cloudflare' }}
+    name: Release
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: write
      pull-requests: write
      issues: read
```

Reply:
You think this is needed, the Wrangler-SDK repo doesn't have them set up?

Reply:
workers-sdk grants all workflows read/write access at the repo level. this repo doesn't, and it would be better to explicitly scope permissions.